examples/models/metadata/graph-metadata/combiner.yaml

```diff
@@ -9,39 +9,60 @@ spec:
   - componentSpecs:
     - spec:
         containers:
-        - image: seldonio/metadata-generic-node:0.2
+        - image: seldonio/metadata-generic-node:0.3
           name: node-combiner
           env:
           - name: MODEL_METADATA
             value: |
-              {"name": "node-combiner",
-               "versions": ["generic-node/v0.2"],
-               "platform": "seldon",
-               "inputs": [
-                   {"name": "input-1", "datatype": "BYTES", "shape": [1, 20]},
-                   {"name": "input-2", "datatype": "BYTES", "shape": [1, 30]}
-               ],
-               "outputs": [{"name": "output", "datatype": "BYTES", "shape": [1]}]}
-        - image: seldonio/metadata-generic-node:0.2
+              ---
+              name: node-combiner
+              versions: [ generic-node/v0.3 ]
+              platform: seldon
+              inputs:
+              - messagetype: tensor
+                schema:
+                  names: [one-output]
+              - messagetype: tensor
+                schema:
+                  names: [two-output]                    
+              outputs:
+              - messagetype: tensor
+                schema:
+                  names: [combiner-output]
+        - image: seldonio/metadata-generic-node:0.3
           name: node-one
           env:
           - name: MODEL_METADATA
             value: |
-              {"name": "node-one",
-               "versions": ["generic-node/v0.2"],
-               "platform": "seldon",
-               "inputs": [{"name": "input", "datatype": "BYTES", "shape": [1, 10]}],
-               "outputs": [{"name": "output", "datatype": "BYTES", "shape": [1, 20]}]}            
-        - image: seldonio/metadata-generic-node:0.2
+              ---
+              name: node-one
+              versions: [ generic-node/v0.3 ]
+              platform: seldon
+              inputs:
+              - messagetype: tensor
+                schema:
+                  names: [common-input]
```

Comment:
What would common-input be?

Reply:
Just wanted to indicate that it is same format of the input for both models behind combiner.

Reply:
Added shapes and improved naming in whole notebook.